"Could not find a declaration file for module 'xml2js'" when import junti2json with TypeScript

When use junit2json with TypeScript, it cannot build by types error.

```
node_modules/junit2json/dist/index.d.ts:1:20 - error TS7016: Could not find a declaration file for module 'xml2js'. '/Users/kesin/github/CIAnalyzer/node_modules/xml2js/lib/xml2js.js' implicitly has an 'any' type.
  Try `npm install @types/xml2js` if it exists or add a new declaration (.d.ts) file containing `declare module 'xml2js';`
```

It will be fix moving "@types/xml2js" from devDependencies to dependencies.

--- a/package.json
+++ b/package.json
@@ -33,13 +33,13 @@
   },
   "homepage": "https://github.com/Kesin11/ts-junit2json#readme",
   "dependencies": {
+    "@types/xml2js": "0.4.5",
     "xml2js": "0.4.23",
     "yargs": "15.3.1"
   },
   "devDependencies": {
     "@types/jest": "25.2.3",
     "@types/node": "14.0.11",
-    "@types/xml2js": "0.4.5",
     "@types/yargs": "15.0.5",
     "jest": "26.0.1",
     "jest-junit": "10.0.0",

